src/zeit/content/volume/browser/form.py

```diff
@@ -54,25 +49,29 @@ def __init__(self, context, request):
 
         """
         super(Base, self).__init__(context, request)
-        source = zeit.content.volume.interfaces.VOLUME_COVER_SOURCE(
+        covers = zeit.content.volume.interfaces.VOLUME_COVER_SOURCE(
             self.context)
-        for name in source:
-            field = zope.schema.Choice(
-                title=source.title(name), required=False,
-                source=zeit.content.image.interfaces.imageGroupSource)
-            field.__name__ = name
-            field.interface = zeit.content.volume.interfaces.IVolumeCovers
-            self.form_fields += zope.formlib.form.FormFields(field)
-
-        self.field_groups += (gocept.form.grouped.Fields(
-            _('Covers'), tuple(source), css_class='column-right'),)
+        if getattr(self.context, 'product', None) is not None:
```

Comment:
Hier hab ich nen Kommentar vergessen "in der Addform hat der context noch kein Product, seufz"